### PR TITLE
Update to WeatherFlow documentation with networking notes

### PR DESCRIPTION
**Pull request by jeeftor**
## Proposed change

Added some notes about networking requirements for the integration.

https://github.com/home-assistant/core/issues/101423#issuecomment-1759472055

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by shubhalgupta** [comment]
can you please assign me this issue

File: source/_integrations/weatherflow.markdown [hunk 1]
```diff
@@ -56,3 +56,7 @@ Additionally the following diagnostic sensors are available:
 - Battery voltage
 - Signal strength
 - Uptime
+
+## Networking Notes
+
+This integration relies on the ability for Home Assistant to receive `UDP` traffic on port `50222`. If you have a more complex network setup utilizing either VLANs or multiple subnets - you may run into trouble.
```
Comment (frenck) [inline source/_integrations/weatherflow.markdown]:
```suggestion
This integration relies on the ability of Home Assistant to receive `UDP` traffic on port `50222`. You may run into trouble if you have a more complex network setup utilizing either VLANs or multiple subnets.
```

File: source/_integrations/weatherflow.markdown [hunk 2]
```diff
@@ -56,3 +56,7 @@ Additionally the following diagnostic sensors are available:
 - Battery voltage
 - Signal strength
 - Uptime
+
+## Networking Notes
```
Comment (frenck) [inline source/_integrations/weatherflow.markdown]:
```suggestion
## Networking notes
```

**Review by frenck — APPROVED** [review]
Thanks, @jeeftor 👍 

../Frenck

**Pull request merged**